.github/workflows/go.yml

```diff
@@ -0,0 +1,31 @@
+name: Go
+
+on:
+  push:
+    branches: [ master ]
+  pull_request:
+    branches: [ master ]
+
+jobs:
+
+  build:
+    name: Build
+    runs-on: ubuntu-latest
+    steps:
+
+    - name: Set up Go 1.13
+      uses: actions/setup-go@v1
+      with:
+        go-version: 1.13
+      id: go
+
+    - name: Check out code onto GOPATH
+      uses: actions/checkout@v2
+      with:
+        fetch-depth: 1
+        path: /home/runner/work/sources-for-knative/sources-for-knative/go/src/github.com/vmware-tanzu/sources-for-knative
```

Comment:
I bet you can change this to `$GOPATH/src/github.com/vmware-tanzu/sources-for-knative`

Reply:
from:  https://github.com/mvdan/github-actions-golang, you can use

```    - name: Checkout code
      uses: actions/checkout@v2
      with:
        path: ./src/github.com/${{ github.repository }}```